### ISSUE TITLE
planning to share pretrained weights?

### ISSUE THREAD

**Issue by keivanB** [jacobzhaoziyuan/AMR-Loss#2]
Hi, 

I would like to evaluate your model on our dataset. Are you planning to share any pre-trained weights publicly?
